sample/index.html

```diff
@@ -51,6 +56,16 @@ <h2>Sample</h2>
 			} ),
 			document.getElementById( 'root' )
 		);
+
+		document.getElementById( 'readOnly' ).addEventListener( 'click', function() {
+			window.editor.isReadOnly = !window.editor.isReadOnly;
```

Comment:
What if the editor isn't available here? 😄

Reply:
I moved this code to `onInit` component's callback.